frontend/src/components/MeetingRoom/ReportIssue/FeedbackForm/FilePicker/FilePicker.tsx

```diff
@@ -92,14 +93,22 @@ const FilePicker = ({
         )}
         {!imageSrc ? (
           <>
-            <Button
-              sx={{ width: '100%', textTransform: 'none', mb: 1 }}
-              variant="outlined"
-              component="label"
-              onClick={processScreenshot}
-            >
-              Capture screenshot
-            </Button>
+            {!isMobile() && (
+              // The screenshot capture relies on the getDisplayMedia browser API which is unsupported on mobile devices
```

Comment:
Maybe we can use `OT.checkScreenSharingCapability()` instead?

Reply:
Let's keep as-is since this utility is a little simpler 🙏